tools/common/schema/types.go

```diff
@@ -84,6 +84,8 @@ const (
 	CLIOptDatabase = "database"
 	// CLIOptPluginName is the cli option for plugin name
 	CLIOptPluginName = "plugin"
+	// CLIOptConnectAttributes is the cli option for connect attributes (key/values via a url query string)
+	CLIOptConnectAttributes = "connect-attributes"
```

Comment:
@venkat1109 I went with your previous suggestion of allowing arbitrary connect attributes: in this case it is via a string in the format of `key1=value1&key2=value2`